bofire/data_models/types.py

```diff
@@ -59,3 +68,5 @@ def validate_unique(uniques: List[str]) -> List[str]:
 TDiscreteVals = Annotated[List[float], Field(min_length=1)]
 
 TInputTransformSpecs = Dict[str, Union[CategoricalEncodingEnum, AnyMolFeatures]]
+
+TIntPowerOfTwo = Annotated[PositiveInt, AfterValidator(validate_power_of_two)]
```

Comment:
Why does this have a `T` as prefixt? Why do they all have `T`-prefixes? It is also `PositiveInt` instead of `TPositiveInt`.

Reply:
Benni started it, I just corrected it now here. I think it is also at other places. But should be tidied up in a seperate PR.